internal/controller/components/workbenches/monitoring/workbenches-prometheusrules.tmpl.yaml

```
@@ -0,0 +1,177 @@
+            message: 'ODH notebook controller is down!'
+            summary: ODH notebook controller pod is not running
+          expr: absent(up{job=~'ODH Notebook Controller Service Metrics'})
+          for: 5m
+          labels:
+            severity: warning
+            namespace: redhat-ods-applications
+
+      
+      - name: SLOs-probe_success_workbench
+        rules:
+        - alert: RHODS Jupyter Probe Success Burn Rate
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)